test/functional/test_runner.py

```diff
@@ -214,7 +214,7 @@ def main():
                                      epilog='''
     Help text and arguments for individual test script:''',
                                      formatter_class=argparse.RawTextHelpFormatter)
-    parser.add_argument('--combinedlogslen', '-c', type=int, default=0, help='print a combined log (of length n lines) from all test nodes and test framework to the console on failure.')
+    parser.add_argument('--combinedlogslen', '-c', type=int, default=0, metavar='n', help='print a combined log (of length n lines) from all test nodes and test frameworks to the console on failure.')
```

Comment:
```suggestion
    parser.add_argument('--combinedlogslen', '-c', type=int, default=0, metavar='n', help='print a combined log (of length n lines) from all test nodes and test framework to the console on failure.')
```

There is only one test framework

Reply:
Realized the whole sentence was a bit complicated, so I rephrased slightly. Hopefully more readable now. 
@MarcoFalke